tutordiscovery/__about__.py

```diff
@@ -1 +1 @@
-__version__ = "13.0.0"
+__version__ = "13.0.1"
```

Comment:
Please don't bump the version in this commit. I'll release a new version myself.